android/native/jni/Android.mk

```diff
@@ -207,6 +224,41 @@ LOCAL_SRC_FILES := \
 	../../src/voxel.cpp                          \
 	../../src/voxelalgorithms.cpp
 
+# Built-in Lua
+ifdef USE_BUILTIN_LUA
+	LOCAL_SRC_FILES += \
+		../../lib/lua/src/lapi.c \
```

Comment:
should work
```suggestion
		$(wildcard ../../lib/lua/src/*.c) \
```

Reply:
that would drag in lua.c and luac.c which have their own main entrypoints and end up breaking compilation, I had that issue when trying to compile android with bundled lua as well and I ended up removing those files since they aren't used when building lua as a library anyways